aepp/exportDatasetToCloudStorage.py

```diff
@@ -0,0 +1,345 @@
+        else:
+            dataflow_id = self.createDataFlow(dataset_id, compression_type, data_format, export_path, on_schedule, config_path)
+        self.createFlowRun(dataflow_id, dataset_id, on_schedule, initial_delay)
+
+    def createDataFlow(
+            self,
+            dataset_id: str,
+            compression_type: str,
+            data_format: str,
+            export_path: str,
+            on_schedule: bool,
+            config_path: str):
+        """
+        Create a data flow as it is not saved in config
+        Arguments:
+          dataset_id : REQUIRED : The dataset that needs to be exported
+          compression_type: REQUIRED : define compression type to use when saving data to file. Possible values:
+            gzip
+            none
+          data_format : REQUIRED : define the data format . Possible values:
+            json
+            parquet
+          export_path : REQUIRED : define the folder path in your dlz container
+          on_schedule: REQUIRED: define whether you would like to have the export following a fixed schedule or not
+        """
+        base_connection_id = self.createBaseConnection()
+        source_connection_id = self.createSourceConnection(dataset_id)
+        target_connection_id = self.createTargetConnection(base_connection_id, compression_type, data_format, export_path)
+        dataflow_id = self.createFlow(source_connection_id, target_connection_id, on_schedule)
+        #save dataflow_id to config
+        Utils.save_field_in_config("Platform", "dataflow_id", dataflow_id, config_path)
+        return dataflow_id
+
+    def createBaseConnection(self) -> str:
+        """
+        Create a base connection for Data Landing Zone Destination
+        Returns:
+            base_connection_id(str)
+        """
+        base_connection_res = self.flow_conn.createConnection(data={
+            "name": f"[CMLE][Week2] Base Connection to DLZ created by {self.username}",
+            "auth": None,
+            "connectionSpec": {
+                "id": self.DLZ_CONNECTION_SPEC_ID,
+                "version": "1.0"
+            }
+        })
+        try:
+            base_connection_id = base_connection_res["id"]
+            self.logger.info("baseConnectionId " + base_connection_id + "has been created")
+            return base_connection_id
+        except KeyError:
+            raise RuntimeError(f"Error when creating base connection: {base_connection_res}")
+
+    def createSourceConnection(self, dataset_id: str) -> str:
+        """
+        Create a source connection for Data Landing Zone Destination
+        Arguments:
+          dataset_id : REQUIRED : The dataset that needs to be exported
+        Returns:
+          source_connection_id(str)
+        """
+        source_res = self.flow_conn.createSourceConnectionDataLake(
+            name=f"[CMLE][Week2] Featurized Dataset source connection created by {self.username}",
```

Comment:
should we make this text configurable and dynamic

Reply:
I believe we are following the consistent entity naming convention in CMLE notebooks.